### ISSUE TITLE
Scheduler with gradient clipping

### ISSUE THREAD

**Issue by JinraeKim** [FluxML/ParameterSchedulers.jl#54]
### Motivation and description

I think most implementations would require gradient clipping.
For example, `Step` (exp-decay with step size) with gradient clipping setting a lower bound of the learning rate.

These were common in previous versions of Flux.
So I think it would be very useful if such functionality is provided by default with keyword arguments.

If it is already implemented, I don't think the current documentation describes it well as I couldn't find it in the docs.

**Comment by ToucheSir** [comment]
https://discourse.julialang.org/t/learning-rate-scheduler-with-the-new-interface-of-flux/107142 is a cross-post of this, so closing the GH issue in favour of the more appropriate platform for questions.
